crates/typst-library/src/layout/page.rs

```diff
@@ -182,20 +169,15 @@ pub struct PageElem {
     #[ghost]
     pub columns: NonZeroUsize,
 
-    /// The page's background fill.
+    /// ページ背景の塗り潰し。
     ///
-    /// Setting this to something non-transparent instructs the printer to color
-    /// the complete page. If you are considering larger production runs, it may
-    /// be more environmentally friendly and cost-effective to source pre-dyed
-    /// pages and not set this property.
+    /// これを非透明なものに設定すると、プリンターにページ全体を着色するよう指示します。
+    /// 印刷規模が大きい場合は、このプロパティを設定せず、あらかじめ染色された用紙を調達したほうが環境にも優しくコスト効率が高いでしょう。
```

Comment:
本筋ではありませんし、雰囲気でもわかるとは思いますが、「コスト効率が高い」というところで「削減」を読み落としてしまうと「コスト→高い」とつながってしまい混乱するかもしれません。「コスト削減」や「経済効果が高い」という方が誤解が生まれないかもしれません。

```suggestion
    /// 印刷規模が大きい場合は、このプロパティを設定せず、あらかじめ染色された用紙を調達したほうが環境にも優しくコスト削減もできるでしょう。
```